tensorflow/core/util/gpu_kernel_helper.h

```diff
@@ -45,6 +45,23 @@ limitations under the License.
 #define GPU_AXIS_KERNEL_LOOP(i, n, axis) \
   for (int i : ::tensorflow::GpuGridRange##axis<int>(n))
 
+#if TENSORFLOW_USE_ROCM
+
+#define cub hipcub
+#define cudaSuccess hipSuccess
+#define cudaGetErrorString hipGetErrorString
+#define cudaStream_t hipStream_t
+#define cudaGetLastError hipGetLastError
+#define cudaError hipError
+
+#define CUDA_1D_KERNEL_LOOP(i, n)                                   \
+    for (size_t i = blockIdx.x * blockDim.x + threadIdx.x; i < (n); \
+        i += blockDim.x * gridDim.x)
+
+#define GetCudaStream(context) context->eigen_gpu_device().stream()
```

Comment:
`GetGpuStream`

Reply:
Did you also remove the GetCudaStream macro when you removed cuda_kernel_helper.h ?

Reply:
I believe so. You might want to double check though.